mlir/include/mlir/Dialect/MIOpen/gridwise_gemm_params.h

```diff
@@ -594,7 +601,7 @@ class PopulateParams : public PopulateParamsBase {
                                        derived);
   }
 
-  int64_t calculateGemmCDestDataPerWrite(ConvolutionContext &ctx) {
+  int64_t calculateGemmCDestDataPerWrite(InitParamsNonXDL *param, ConvolutionContext &ctx) {
```

Comment:
Could you pass by const reference so as to be consistent with the rest of coding style? Also applies to `CalculateBlockGemmPerformanceParameters `